mtp_cashbook/templates/cashbook/transaction_batch_list.html

```diff
@@ -43,7 +43,7 @@
           <td class="numeric">&pound;{{ transaction.amount|currency }}</td>
           <td>{{ transaction.sender }}</td>
           <td class="check">
-            <input type="checkbox" name="{{ form.transactions.html_name }}" value="{{ transaction_pk }}" id="check-{{ transaction.id }}" {% if transaction_pk|to_string in form.transactions.value %}checked{% endif %}>
+            <input type="checkbox" name="{{ form.transactions.html_name }}" value="{{ transaction_pk }}" id="check-{{ transaction.id }}" {% if transaction_pk|to_string in form.transactions.value %}checked{% endif %} class="js-RunningTotal-item" data-amount="{{ transaction.amount|currency }}">
```

Comment:
Can we break this line up a bit, @teneightfive? Maybe split between attributes or something?